src/urllib3/response.py

```diff
@@ -3,7 +3,9 @@
 import typing
 import zlib
 from contextlib import contextmanager
+from http.client import HTTPResponse as ClientHTTPResponse
```

Comment:
Let's import this as `_HttplibHTTPResponse`

Reply:
Done